src/sentry/api/serializers/models/project.py

```diff
@@ -100,95 +99,102 @@ def get_access_by_project(self, item_list, user):
         return result
 
     def get_attrs(self, item_list, user):
-        project_ids = [i.id for i in item_list]
-        if user.is_authenticated() and item_list:
-            bookmarks = set(
-                ProjectBookmark.objects.filter(user=user, project_id__in=project_ids).values_list(
-                    "project_id", flat=True
+        from sentry import features
+        from sentry.features.base import ProjectFeature
```

Comment:
Nit: Can we import these at the top of the file? Or do we have a circular import issue?

Reply:
Maybe; I'll check. I'll move it if possible.

Reply:
Fixed.